src/components/Notification/NotificationDrawer/NotificationDrawerTitle.js

```diff
@@ -0,0 +1,61 @@
+import ClassNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from '../../Icon';
+
+const NotificationDrawerTitle = ({
+  expandable,
+  onCloseClick,
+  onExpandClick,
+  title,
+  children,
+  className,
+  ...props
+}) => {
+  const classes = ClassNames('drawer-pf-title', className);
+
+  if (expandable) {
+    return (
+      <div className={classes} {...props}>
+        <a className="drawer-pf-toggle-expand" onClick={onExpandClick} />
+        <a className="drawer-pf-close" onClick={onCloseClick}>
```

Comment:
rather than duplicating all but this, make this piece conditional:
```
{expandable && (
        <a className="drawer-pf-toggle-expand" onClick={onExpandClick} />
)}
```